etc/bin/run.sh

```diff
@@ -30,7 +30,10 @@ KERNEL_ASSEMBLY=`(cd ${PROG_HOME}/lib; ls -1 toree-kernel-assembly-*.jar;)`
 
 # disable randomized hash for string in Python 3.3+
 export PYTHONHASHSEED=0
+# The SPARK_OPTS values during installation are stored in TOREE_SPARK_OPTS. This allows values to be specified during
+# install, but also during runtime.
+export SPARK_OPTS="${TOREE_SPARK_OPTS} ${SPARK_OPTS}"
```

Comment:
Putting things in the kernel spec, as you state below, does not work. When the env vars from the kernel spec are exported, the substitution does not happen and `SPARK_OPTS` would be literally `<command line args during install> ${SPARK_OPTS}`. I tried different approaches, in `run.sh`, with evaluating `SPARK_OPTS` to expand the variable, but at that point the variable is set and there would be no expansion to what the user exports. This ultimately coming down to how Jupyter is spawning the kernels. We would need to make a change there to evaluate and expand the variables.